test/core/tsi/transport_security_test_lib.c

```diff
@@ -577,6 +573,8 @@ void tsi_test_fixture_init(tsi_test_fixture *fixture) {
   fixture->bytes_read_from_server_channel = 0;
   fixture->test_unused_bytes = true;
   fixture->has_client_finished_first = false;
+  fixture->notification = gpr_zalloc(sizeof(tsi_test_notification));
```

Comment:
Could just make this structure a direct member of the fixture and avoid the additional dynamic allocation.

In fact, you could just make the members of `tsi_test_notification` direct members of `tsi_test_fixture` and avoid having a separate struct altogether.

Reply:
I just submitted a new commit that make the members of tsi_test_notification direct members of tsi_test_fixture.